docs/dev/run_the_etl.rst

```diff
@@ -160,7 +160,8 @@ a window that looks like this:
 Click the hamburger button in the upper left to view the definitions,
 assets and jobs.
 
-**Cloning the FERC databases**
+**Cloning the FERC databases:**
```

Comment:
If the things being defined are sub-sub-headers I think they'll stand out more as separate items/concepts, and they will also appear in the navigation in the upper-right corner of the page.